src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import dao.impl.ReaderServiceImpl;
+import dao.impl.WriterServiceImpl;
+import java.util.HashMap;
+import java.util.Map;
+import service.CreateReportService;
+import service.ProcessData;
+import service.impl.CreateReportServiceImpl;
+import service.impl.ProcessDataImpl;
+import strategy.ActivitiesStrategyImpl;
+import strategy.activities.ActivitiesHandler;
+import strategy.activities.BalanceHandler;
+import strategy.activities.PurchaseHandler;
+import strategy.activities.ReturnHandler;
+import strategy.activities.SupplyHandler;
+
+public class Main {
+    private static Map<String, ActivitiesHandler> activitiesHandlerMap = new HashMap<>();
+
+    private static CreateReportService createReportService = new CreateReportServiceImpl(
+            new WriterServiceImpl("./src/main/resources/report.csv"));
+
+    private static ProcessData processData = new ProcessDataImpl(
+            new ReaderServiceImpl("./src/main/resources/database.csv"),
+            new ActivitiesStrategyImpl(activitiesHandlerMap));
```

Comment:
let's create constants from paths, and we will create objects in the main method